core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -70,6 +72,7 @@ class ReligionManager {
 
     fun endTurn(faithFromNewTurn: Int) {
         storedFaith += faithFromNewTurn
+        storedFaith += getFaithFromCityStates()
```

Comment:
so now this is no longer necessary, since the values in Eras.son are already in the faithFromNewTurn :)

Reply:
Is what i have done okay?